commit 7c292df075fcd845f8ca3a90dacff29b32df62d5 (refs/changes/06/406/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-01T10:41:50-07:00 (3 years, 1 month ago)
    
    MANTA-2863 manta zone setup scripts need to wait for config-agent to start

diff --git a/deps/manta-scripts b/deps/manta-scripts
index 6c96273..cd72e80 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit 6c96273f7281df37ba60e4cb8b4cc4dd27ff1f1e
+Subproject commit cd72e801576de5a4b1aec35a7f79409d3f1eebb5
